components/inputs/input-color.js

```diff
@@ -104,7 +104,13 @@ class InputColor extends PropertyRequiredMixin(FocusMixin(FormElementMixin(Local
 			 * REQUIRED: Label for the input, comes with a default value for background & foreground types.
 			 * @type {string}
 			 */
-			label: { type: String },
+			label: {
+				type: String,
+				required: {
+					dependentProps: ['type'],
+					validator: (_value, elem, hasValue) => elem.type !== 'custom' || hasValue
```

Comment:
We now need to pass `elem` to the validator and message options since `properties` is `static` and doesn't have access to `this`.